### PR TITLE
✨ Add support for .NET Aspire

### PR DESCRIPTION
**Pull request by jasontaylordev**
Added support for .NET Aspire. Create a new solution and include `--use-aspire`.
Requested in #1273

### CODE REVIEW

File: src/AppHost/Program.cs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddProject<Projects.Web>("web");
+
+builder.Build().Run();
```
Comment (melotic) [inline src/AppHost/Program.cs:5]: Should we add a SQL Server component here? The template uses a LocalDb by default.

Comment (jasontaylordev) [reply src/AppHost/Program.cs]: Thanks for the suggestion. I've updated it to include SQL Server integration.

Comment (pramodkumarjaiswal) [reply src/AppHost/Program.cs]: Hi, I'm trying to setup a new WebAPI solution using "--use-aspire" but it's not working, and says -is not a valid option.
Could you please suggest the right commend to create a new Aspire solution from template?

Comment (melotic) [reply src/AppHost/Program.cs]: > Hi, I'm trying to setup a new WebAPI solution using "--use-aspire" but it's not working, and says -is not a valid option. Could you please suggest the right commend to create a new Aspire solution from template?

This PR is not merged and not released yet, so you'd have to build and install the templates from this branch to use the flag.

Comment (pramodkumarjaiswal) [reply src/AppHost/Program.cs]: I tried below and it gave me same error. Am I missing something?

Comment (pramodkumarjaiswal) [reply src/AppHost/Program.cs]: pramodjaiswal@192 CleanArchitecture % dotnet new ca-sln --use-aspire -cf None -o OrchestrationEngine
Error: Invalid option(s):
--use-aspire
   '--use-aspire' is not a valid option

Comment (jasontaylordev) [reply src/AppHost/Program.cs]: Sorry, that will not work until this PR is merged and a NuGet package is published. I'll try to get it done today.

**Pull request merged**